### PR TITLE
[bug 1142093] Replace jingo-minify with django-pipeline

### PR DESCRIPTION
**Pull request by rehandalal**
f?

DO NOT MERGE until bug #1170183 is fixed. (https://bugzilla.mozilla.org/show_bug.cgi?id=1170183)

### CODE REVIEW

File: fjord/base/static_utils.py [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-
-    This is super lame!
-
-    It is specifically for when DEBUG = True because jingo-minify puts
-    compiled files in STATIC_ROOT. gah!
-    """
-    storage = StaticFilesStorage
-
-    def list(self, ignore_patterns):
-        return []
```
Comment (willkg) [inline fjord/base/static_utils.py]: NOOOOOOO!!!!!!!

**Comment by willkg** [comment]
This looks ok as far as I can tell.

1. What do we need to change in the docs?
2. What do we need to change to `vagrant/vagrant_provision.sh`?
3. What do we need to change on the servers and do we need to do it before this lands?

**Comment by willkg** [comment]
I haven't heard anything on this for 6 days. Where's it at? Do you need help with things?

**Comment by willkg** [comment]
@rehandalal Don't worry about the vagrant side of things. I can figure that out or throw it into a bug for someone else to chew on.

**Comment by willkg** [comment]
Can you rebase this against master thus picking up all the changes in the last 3 weeks?

**Comment by rehandalal** [comment]
yay ^

**Comment by willkg** [comment]
Adding a note so I remember:

DO NOT MERGE until bug #1170183 is fixed. (https://bugzilla.mozilla.org/show_bug.cgi?id=1170183)

**Comment by willkg** [comment]
This branch is pretty old and a lot has changed. I'm going to close this PR out, grab @rehandalal's branch (with his permission) and then finish up the work. I'll create a new PR then.

**Pull request closed without merging**